client/checkout/classic/style.scss

```diff
@@ -41,32 +41,48 @@
 
 	&.wc_payment_methods,
 	&.woocommerce-PaymentMethods {
-		li.payment_method_woocommerce_payments {
-			display: grid;
-			grid-template-columns: 0fr 0fr 1fr;
-			grid-template-rows: max-content;
-
```

Comment:
I originally used grid on the `li` element to prevent layout issues with themes that don't hide the `input[type=radio]` element and replace it with a custom `::before` icon in the label. 

If you switch to a theme like twenty-twentyfour you'll see that the layout is broken due to this.

![image](https://github.com/user-attachments/assets/b6ddbafd-5a71-40c6-8384-cf23e5ad92ae)

This is how it looks like in develop.
![image](https://github.com/user-attachments/assets/90b4fdc0-3941-4357-9e1a-f08981cf271b)